iroha-cpp/services/Query.hpp

```diff
@@ -0,0 +1,39 @@
+#ifndef QUERY_HPP
+#define QUERY_HPP
+
+#include "LibsAndClassDeclarations.h"
+
+#include "crypto/keypair.hpp"
+#include "queries.pb.h"
+
+
+namespace IROHA_CPP
+{
+
+class Query
+{
+    std::string creator_;
+    uint64_t counter_;
+    uint64_t local_time_;
+    iroha::protocol::Query queryProto;
```

Comment:
according to convention:
```suggestion
    iroha::protocol::Query queryProto_;
```

Reply:
Cleaned up the code with iroha's convention.